.github/CODEOWNERS

```diff
@@ -3,4 +3,4 @@
 
 * @dotnet/roslyn-analysis
 
-eng/SourceBuild* @dotnet/source-build-internal
+eng/SourceBuild* @dotnet/source-build
```

Comment:
Do we need write permissions?

Reply:
yes, I noted it in the PR description that repo admins need to grant write permissions.